index.rst

```diff
@@ -1,57 +1,240 @@
+
+The ATPneumatics is the component is charge of regulating the pressure for the
+pneumatics actuators. It is important to keep a couple things in mind when it
+comes to how this component works. To start with, it is not possible to set
+the pressure to zero. There are 3 valves involved when it comes to setting the
+M1 support pressure; master air supply, instrument air supply and M1 air
+valves. Once all 3 valves are opened there will a small back pressure applied
+to M1 even if it is commanded to zero. It is important to have that in mind
+since it will impact the way we measure the optimum pressure.
+
+The ATAOS is the component in charge of storing the look-up table and setting
+the M1 pressure in the ATPneumatics depending on the elevation. It works by
+listening to the position of telescope, published by the ATMCS, and the current
+M1 support pressure, published by the ATPneumatics. The elevation is used to
+calculate the expected pressure, according to a polynomial equation. If the
+computed value changes by more than a configurable threshold compared to the
+current pressure, it will send a command to the ATPneumatics to update the M1
+support pressure.
+
+This document describes the procedure to measure the optimum M1 pressure for
+different telescope elevations and fit the data with a polynomial equation
+to obtain the coefficients required by the ATAOS component. We also describe
+how to load the new coefficients into the ATAOS.
+
+
+Measuring optimum M1 support pressure for different elevations
+==============================================================
+
+.. important::
+   During this process it is important to make sure one does not lift the
+   mirror from the hard points. This will happen if the value measured by
+   the load cell gets close to zero. In this case, it is important to lower
+   the mirror all the way back, setting the pressure to zero, close the
+   M1 valve in the ATPneumatics, slew the telescope to the park position
+   (:math:`\mathrm{elevation} = 86^\circ`), and restart the process.
+
+As mentioned above, the support system contain 3 hard points and 21 actuators,
+all have the same contact area with the mirror. One of the hard points (the
+load cell) contain a "digital scale" that measures the load (how much weight)
+it is supporting.
+
+In principle, to minimize distortion in the mirror shape we must make sure
+the load is evenly distributed between all the 24 actuators. The first step in
+the process is determining what it expected "optimum load", as measured by the
+hard point load scale mentioned above. When there is no pressure in the
+pneumatic system all the weight of the mirror will be supported by the 3 hard
+points. That means the "total load" for a given elevation is 3x the one
+measured by the load cell. Consequently, the "optimum load" will be the "total
+load" divided by 24, the number of actuators.
+
+.. math:: \mathrm{optimum\_load} = \frac{\mathrm{load\_cell} \times 3}{24}
```

Comment:
see you applied a label here which gives a number in a terrible format. I tried messing with the math:: and label stuff last night for quite some time for a technote i'm writing and eventually gave up... sounds like you've hit the same issue 😞

Reply:
Yes... this has probably something to do with the Sphinx theme.. maybe something Jonathan can take a look.